src/main/java/tc/oc/item/Items.java

```diff
@@ -2,12 +2,7 @@
 
 import com.google.common.collect.ImmutableSet;
 import com.google.common.collect.Sets;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Should probably not be changing the whole project over to wildcard imports. Wildcard imports are to be avoided, especially when they're gonna create such a big diff that it's gonna be a nightmare to deal with

Reply:
Hmm, there are wild cards in many places in the code.. I think I'll fix that in a separate commit.